_config/features_comparison.yml

```diff
@@ -0,0 +1,241 @@
+features:
+  - name: "ACID"
+    jta: true
+    jts: true
+    xts: true
+    comp: false
+    rts: true
+    stm: true
```

Comment:
Maybe we could add all exceptions bellow the table? The only options to add * in the specific line is to modify name e.g. "ACID*"